packages/cli/commands/create.js

```diff
@@ -200,6 +211,62 @@ exports.handler = async options => {
       createTemplate(name, dest, templateType);
       break;
     }
+    case TYPES['api-sample']: {
+      const filePath = path.join(dest, name);
+      if (fs.existsSync(filePath)) {
+        const { overwrite } = await overwriteSamplePrompt(filePath);
+        if (overwrite) {
+          const removingSpinner = ora(`Removing existing ${filePath} folder`);
+          removingSpinner.start();
+          fs.rmdirSync(filePath, { recursive: true });
+          removingSpinner.stop();
+        } else {
+          return;
+        }
+      }
+      const downloadSpinner = ora('Downloading API samples config');
+      downloadSpinner.start();
+      const samplesConfig = await downloadConfig(
+        PROJECT_REPOSITORIES[assetType],
+        'samples.json'
+      );
+      downloadSpinner.stop();
+      if (!samplesConfig) {
+        logger.error(
+          `Currently there are no samples available, please, try again later.`
+        );
+        return;
+      }
+      const { sampleType, sampleLanguage } = await createApiSamplePrompt(
```

Comment:
Since at this stage, we only are offering a single node sample project. I think we should remove the prompts or remove `api-sample` from `hs create --help`.

Reply:
We're going to add new languages to existing sample app, and create new sample apps shortly, so that's why we want to leave prompts. So, will remove api-sample from help temporarily.

Reply:
removed api-sample from hs create help